lib/mock-axios-types.ts

```diff
@@ -140,4 +140,4 @@ export interface AxiosMockQueueItem {
  * Axios object can be called like a function,
  * that's why we're defining it as a spy
  */
-export type AxiosMockType = AxiosFn & AxiosAPI & AxiosMockAPI;
+export type AxiosMockType = AxiosFn & AxiosAPI & AxiosMockAPI & jest.Mock;
```

Comment:
I think we can provider better type definitions here:
As `jest.Mock` is actual [generic](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jest/index.d.ts#L842), we can provide at least one type argument:

```typescript
jest.Mock<SyncPromise, [any?]>;
```